packages/react-native/React/Base/RCTConvert.mm

```diff
@@ -16,6 +16,8 @@
 #import "RCTParserUtils.h"
 #import "RCTUtils.h"
 
+#import <react/renderer/graphics/ColorComponents.h>
```

Comment:
Not for us: this might require us to update the `podspec` to include react-graphics, if that's not like this already. Not a problem, but just mentioning not to forget about it if we see failures in CI due to Dynamic Frameworks.